src/components/Members.jsx

```diff
@@ -18,19 +22,20 @@ function Members() {
         <h3>Meet the Admins</h3>
         <div className="row">
           {admin.map((member) =>
-            <div key={member.name}>
-              <img className='admin' src={member.imgUrl} alt={`beautiful headshot of ${member.name}.`} />
-              <h4>{member.name}</h4>
+            <div key={member.id}>
```

Comment:
Good job catching this and adding the key. I'm guessing you maybe noticed a warning in the browser console? I'm usually good at remembering to test with my console open but have been forgetting about it recently.. 😑

Reply:
Happens to the best of us!